### PR TITLE
AB#39081 add app permissions sample query: POST teams channels

### PR DESCRIPTION
**Pull request by acchiang**
Copied from the other POST teams channels sample query, but with new generated GUID and added information in the tip to ensure the user knows to switch to user mode in order to get Teams IDs. 
Also added a space between the link and the period in the tip so that the period is not automatically added to the autofill input box when a user clicks on it.

### CODE REVIEW

**Review by omersayshi — APPROVED** [review]

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -1418,7 +1418,7 @@
             }
         ],
         "postBody": "{\r\n   \"displayName\": \"Architecture Discussion\",\r\n   \"description\": \"This channel is where we debate all future architecture plans\"\r\n }",
-        "tip": "This query requires a team id.  To find the team id of teams you belong to, you can run: GET https://graph.microsoft.com/v1.0/me/joinedTeams.",
+        "tip": "This query requires a team id.  To find the team id of teams you belong to, you can run: GET https://graph.microsoft.com/v1.0/me/joinedTeams .",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]: This bug has been patched as per [AD#39035](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_sprints/taskboard/GI21%20-%20Graph%20Explorer/Intern%20GitHub/Y21-S/07?workitem=39035) (which was just merged into main GE)! There is no need to remove the period anymore ^^

```suggestion
        "tip": "This query requires a team id.  To find the team id of teams you belong to, you can run: GET https://graph.microsoft.com/v1.0/me/joinedTeams.",
```

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -2983,6 +2983,22 @@
...
   ],
   "TeamsAppSampleQueries": [
-      
+    {
+        "id": "9d9dc771-9926-4e41-b2e9-26159bb80c85",
+        "category": "Microsoft Teams",
+        "method": "POST",
+        "humanName": "create channel",
+        "requestUrl": "/v1.0/teams/{team-id}/channels",
+        "docLink": "https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/channel_post",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]: Shouldn't we be using beta? (I could be wrong)
```suggestion
        "category": "Microsoft Teams (beta)",
        "method": "POST",
        "humanName": "create channel",
        "requestUrl": "/beta/teams/{team-id}/channels",
        "docLink": "https://docs.microsoft.com/en-us/graph/api/channel-post?view=graph-rest-beta",
```

**Review by ayubun — CHANGES_REQUESTED** [review]
If we don't use the beta tag for some of the queries, they end up in a separate drop-down. Maybe we want this? But I kind of assumed we have been prioritizing the beta queries because that's what I've seen before.

Lmk @omersayshi @acchiang 👀

File: sample-queries/sample-queries.json [hunk 3]
```diff
@@ -2984,44 +2984,61 @@
...
+        "method": "GET",
+        "humanName": "get list of members in team",
+        "requestUrl": "/v1.0/teams/{team-id}/members",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/team-list-members?view=graph-rest-1.0&tabs=http",
+        "tip": "This query requires a team id. To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams.",
+        "skipTest": false
+    },
+    {
+        "id": "fe15123a-507a-45d8-aa29-e5b25f5c0ac7",
+        "category": "Microsoft Teams",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "category": "Microsoft Teams (beta)",
```

**Review by ayubun — APPROVED** [review]

**Comment by ayubun** [comment]
LGTM! Be sure to test it before merging since there have been more commits added 🥺🙏

**Comment by acchiang** [comment]
great callouts @AyuAi!

**Pull request merged**